.evergreen/config/functions.yml

```diff
@@ -152,7 +162,7 @@ functions:
       params:
         script: |
           ${PREPARE_SHELL}
-          ${DRIVERS_TOOLS}/.evergreen/stop-orchestration.sh
+          bash ${DRIVERS_TOOLS}/.evergreen/stop-orchestration.sh
```

Comment:
After reading DRIVERS-2852 I understand that this is a temporary change during migration from `sh` to full `bash`.